app/assets/javascripts/application/codemirror.js

```diff
@@ -4,11 +4,9 @@ var mumuki = mumuki || {};
   function createCodeMirrors() {
     var editors = $(".editor").map(function (index, textarea) {
       var $textarea = $("#solution_content");
-      var builder = new mumuki.editor.CodeMirrorBuilder(textarea);
-      builder.setupEditor();
-      builder.setupOptions($textarea.data('lines'));
-      builder.setupLanguage();
-      return builder.build();
+      return new mumuki.editor.CodeMirrorBuilder(textarea)
+        .setup($textarea.data('lines'))
```

Comment:
This is exactly what I wanted to avoid with the builder: having default arguments in a "big" `setup` method, in order to make explicit what we are actually configurating - in the old days there were more configuration options. 

I think the configuration code from `codemirror.js` or `multifiles-editor.js` should be something like this: 

```javascript
let builder = new mumuki.editor.CodeMirrorBuilder(textarea);
builder.setupMinLines(/*...minLines...*/);
builder.setupLanguage(/*...language...*/);  // or no call at all
builder.build();
```